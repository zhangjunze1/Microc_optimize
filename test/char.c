char a;
void main() {
  char b;
  char c='c';
  a='a';
  b='b';
  print("%c",a);
  print("%c",b);
  print("%c",c);
}
