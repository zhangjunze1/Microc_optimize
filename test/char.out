15 1 24 19 0 7 25 15 1 13 0 1 1 0 99 12 0 0 0 97 12 15 -1 13 0 98 12 15 -1 0 0 11 23 15 -1 13 11 23 15 -1 13 0 1 1 11 23 21 2